### DISCUSSION TITLE
添加天翼云盘出错

### DISCUSSION THREAD

**Discussion by briheart** [AlistGo/alist#2633] [Q&A]
提示啥时间不一致。。
“Failed init storage but storage is already created: Post "https://api.nn.ci/ocr/file/json": x509: certificate has expired or is not yet valid: current time 2022-12-07T16:01:15+08:00 is after 2021-09-30T14:01:15Z”

请问什么问题。谢谢

**Comment by zch11230** [comment]
#477

**Reply by zch11230** [reply]
群晖6吗？我改用的docker
